core/src/main/java/com/nisovin/magicspells/spells/PlayerMenuSpell.java

```diff
@@ -60,6 +60,8 @@ public class PlayerMenuSpell extends TargetedSpell implements TargetedEntitySpel
 	private Subspell spellOnSneakRight;
 	private ModifierSet playerModifiers;
 
+	private static int maxPlayersPerPage = 50;
```

Comment:
It's unnecessary to have it be a private field; it's never modified and its constant. If it weren't static and instead a configurable option, that'd make more sense.